### PR TITLE
Fix the last of the docs warnings

### PR DESCRIPTION
**Pull request by EraYaN**
**Changes**
Fixed some casing and other formatting to make all the missing cref warnings go away.

EDIT: it fixes like 2 errors. locally it emits 0 warnings but CI emits many.

### CODE REVIEW

File: MediaBrowser.Model/MediaBrowser.Model.csproj [hunk 1]
```diff
@@ -13,9 +13,14 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\MediaBrowser.Common\MediaBrowser.Common.csproj" />
```
Comment (EraYaN) [inline MediaBrowser.Model/MediaBrowser.Model.csproj]: Alright this creates a cycle sadly.

File: MediaBrowser.Providers/Tmdb/Models/Search/MovieResult.cs [hunk 2]
```diff
@@ -2,11 +2,11 @@ namespace MediaBrowser.Providers.Tmdb.Models.Search
...
-            /// <summary>
-            /// Gets or sets a value indicating whether this <see cref="TmdbMovieSearchResult" /> is adult.
-            /// </summary>
-            /// <value><c>true</c> if adult; otherwise, <c>false</c>.</value>
-            public bool Adult { get; set; }
+        /// <summary>
+        /// Gets or sets a value indicating whether this <see cref="MovieResult" /> is adult.
+        /// </summary>
+        /// <value><c>true</c> if adult; otherwise, <c>false</c>.</value>
+        public bool Adult { get; set; }
```
Comment (Bond-009) [inline MediaBrowser.Providers/Tmdb/Models/Search/MovieResult.cs]: Indentation seems off.

Comment (EraYaN) [reply MediaBrowser.Providers/Tmdb/Models/Search/MovieResult.cs]: The original file has a 8 space tab and 1 4 space tab, should I reformat the whole file?

Comment (Bond-009) [reply MediaBrowser.Providers/Tmdb/Models/Search/MovieResult.cs]: Yeah, just reformat it so it conforms to our standards.

**Pull request merged**